src/components/month-picker.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+
+import Picker from 'react-month-picker'
+
+import "react-month-picker/css/month-picker.css";
+
+const MonthPicker = () => {
+
+    const currentDate = new Date();
+    const currentMonth = currentDate.getMonth();
+    const currentYear = currentDate.getFullYear();
+
+    const [isVisible, setVisibility] = useState(false);
+    const [selected_month_year, setMonthYear] = useState({ month: currentMonth, year: currentYear });
+
+
+    const monthsList = [
+        "Jan",
+        "Feb",
+        "Mar",
+        "Apr",
+        "May",
+        "Jun",
+        "Jul",
+        "Aug",
+        "Sep",
+        "Oct",
+        "Nov",
+        "Dec"
+    ]
+
+    const range = {
+        min: { month: 1, year: currentYear - 1, },
+        max: { month: currentMonth + 1, year: currentYear, }
+    };
+    const showMonthPicker = event => {
+        setVisibility(true);
+        event.preventDefault();
+    };
+
+    const handleOnDismiss = () => {
+        setVisibility(false);
+    };
+    const getWeeksInMonth = (year, month) => {
+        const weeks = [];
+        const firstDate = new Date(year, month - 1, 1);
+        const lastDate = new Date(year, month, 0);
+        const numDays = lastDate.getDate();
+        console.log(numDays);
```

Comment:
Isn't it better to remove this `console`😁

Reply:
@Bahar-FS consoled for now since it needs to be passed to the component farrah is working on.